manim/logger.py

```diff
@@ -1,12 +1,46 @@
+import configparser
 import logging
+
+from rich.console import Console
 from rich.logging import RichHandler
+from rich.theme import Theme
+
+from .utils.config_utils import config_parser, successfully_read_files
+
+
+def parse_theme(fp):
+    config_parser.read(fp)
+    theme = dict(config_parser["log.color"])
+    customTheme = Theme(theme)
+    return customTheme
+
 
+try:
+    customTheme = parse_theme(successfully_read_files)
+    console = Console(theme=customTheme)
+except KeyError:
+    console = Console()
```

Comment:
Please print/log the error message here as well

Reply:
Do we need to expicitely say people that you don't have a config file create one?

Reply:
"No cfg file found, creating one in <path_to_file>" would be really good

Reply:
Wait, its `logger.py` file and not `cfwriter.py` file. Always I will write, this line `No cfg file found in <path_to_file>`.